lib/hook/consumers/github/sync.ex

```diff
@@ -52,8 +52,8 @@ defmodule Accent.Hook.Consumers.GitHub.Sync do
         assigns: %{
           document: document,
           document_update: %{
-            top_of_the_file_comment: parsed_document.top_of_the_file_comment,
-            header: parsed_document.header
+            top_of_the_file_comment: parsed_document && parsed_document.top_of_the_file_comment,
```

Comment:
maybe we could have something like 
```elixir
document_update: document_update(parsed_document)
....

defp document_update(nil) do
  # And maybe il will make more sense just to return nil instead of a map with nil values ?
  %{top_of_the_file_comment: nil, header: nil}
end
defp document_update(parsed_document) do
  %{top_of_the_file_comment: parsed_document.top_of_the_file_comment, header: parsed_document.header}
end
```